build-tools/packages/build-cli/src/library/git.ts

```diff
@@ -43,6 +44,52 @@ function getVersionFromTag(tag: string): string | undefined {
 	return ver.version;
 }
 
+/**
+ * Get all the versions for a release group. This function only considers the tags in the repo.
+ *
+ * @param git - The git repository.
+ * @param releaseGroup - The release group to get versions for.
+ * @returns List of version details, or undefined if one could not be found.
+ */
+export async function getVersionsFromTags(
+	releaseGroup: IReleaseGroup,
+	git: Readonly<SimpleGit>,
+): Promise<VersionDetails[] | undefined> {
+	const prefix = PackageName.getUnscopedName(releaseGroup.name);
+
```

Comment:
This isn't needed any more, since you don't have package names to deal with. You can just use releaseGroup.name.